### DISCUSSION TITLE
Adding Type Descriptions to aid Online Groups.

### DISCUSSION THREAD

**Discussion by WealthySoul** [code4recovery/12-step-meeting-list#1305] [Ideas]
The online groups that have transferred to our Area 39 from OIAA. Have a description of "Proof of Attendance". This would be nice to have on the MGA. Also there might be more like this that I am not aware of. But since C4R also did the OIAA meeting app, you would have some insight for this.

**Comment by joshreisner** [comment]
yes! we are communicating with GSO to see if they will allow these as types in the app. see the "Proposed" sections in the table here: https://github.com/code4recovery/spec#proposed-new-types

in the meantime you can follow our [instructions to add a new type](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21) locally by adding this to your theme's functions.php:

```php
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'POA' => 'Proof of Attendance',
    ));
}
```

it won't show up in the Meeting Guide mobile apps however
